lib/rules/prefer-object-has-own.js

```diff
@@ -87,6 +87,13 @@ module.exports = {
                                 return null;
                             }
 
+                            const tokenJustBeforeNode = sourceCode.getTokensBefore(node, 1)[0];
```

Comment:
There is a `getTokenBefore` method.

We can also use `includeComments`. if there is comment between, the space it not nessary.